app/src/main/java/com/github/orkest/Model/Post.kt

```diff
@@ -10,5 +10,5 @@ data class Post(var username: String = "",
                 var postDescription: String = "Post Description",
                 var song: Song = Song(),
                 var likes: Int = 0,
-                var comments: List<String> = ArrayList(),
+                var comments: MutableList<Comment> = ArrayList(),
```

Comment:
Since comments are now stored as separate objects in a collection in the database, this property can be removed from the post model